addons/field_rations/functions/fnc_handleEffects.sqf

```diff
@@ -28,13 +28,14 @@ if ((_thirst > 99.9 || {_hunger > 99.9}) && {random 1 < 0.5}) exitWith {
 if !(_player call EFUNC(common,isAwake)) exitWith {};
 
 // Set unit unconscious (chance based on how high thirst/hunger are)
-if ((_thirst > 85 || {_hunger > 85}) && {random 1 < linearConversion [85, 100, _thirst max _hunger, 0.05, 0.1, true]}) exitWith {
-    if (["ace_medical"] call EFUNC(common,isModLoaded)) then {
-        [_player, true, 5, true] call EFUNC(medical,setUnconscious);
-    };
+if (
```

Comment:
logic isn't quite the same (if medical not loaded) but I don't think it matters

Reply:
Probably best to keep the old logic imo.

Reply:
Looking at the code only, this increases the likelihood of players falling down if medical is not loaded (see line 39).
In practice I have no idea what the effect would be.

Reply:
`random 1 < linearConversion [85, 100, _thirst max _hunger, 0.05, 0.1, true]`
so about 5-10% chance of being "protected" from falling over if medical isn't loaded
but the update loop runs every second, so I really doubt anyone will notice this

Reply:
Idc too much either way.
For a future PR it might be interesting to have a function similar to `common`'s `setDead` but for unconscious, which would handle setting people unconscious when using the vanilla system. That being said, I don't know what sort of settings there are for the vanilla system, so I'm a bit hesitant to implement such a function,